spring-cloud-stream-schema/src/test/java/org/springframework/cloud/schema/avro/AvroMessageConverterSerializationTests.java

```diff
@@ -67,9 +67,8 @@
 
 	@Before
 	public void setup() {
-		System.setProperty("--spring.main.allow-bean-definition-overriding", "true");
 		schemaRegistryServerContext = SpringApplication
-				.run(SchemaRegistryServerApplication.class);
+				.run(SchemaRegistryServerApplication.class, "--spring.main.allow-bean-definition-overriding=true");
```

Comment:
DITTO

Reply:
@artembilan There is an isolated case with schema registry that I had to override the bean internally in the unit tests. For the real application, it worked. Since it worked for the sample apps, I think in order to reduce the scope of this PR, we should track this separately before M2. I will create an issue.